lib/vanagon/platform/windows.rb

```diff
@@ -196,6 +172,7 @@ def initialize(name)
         @num_cores = "/usr/bin/nproc"
         @install = "/usr/bin/install"
         @copy = "/usr/bin/cp"
+        @package_type = "msi"
```

Comment:
Hmm, this probably should be nuget for now... Eventually I want this to be MSI, but since we don't have support for building msis yet, projects will have to explicitily set package_type to nuget if they want to start testing this out